src/components/Button/Button.module.css

```diff
@@ -1,14 +1,13 @@
 .base {
-    border-radius: var(--button-border-radius);
-    line-height: var(--button-line-height);
+    border-radius: var(--border-radius-small);
+    line-height: 32px;
     font-size: 14px;
     padding: 0 16px;
     border: none;
     cursor: pointer;
-}
-
-.typePrimary {
-    background-color: var(--button-primary-background);
+    display: flex;
+    flex-direction: row;
+    align-items: center;
```

Comment:
Primary should only have a difference in background color. Move these props to `.base`?